packages/styling/src/styles/getFocusStyle.ts

```diff
@@ -9,12 +10,14 @@ import { ITheme } from '../interfaces/index';
  * @param color - The color for the border.
  * @param position - The positioning applied to the container. Must
  * be 'relative' or 'absolute' so that the focus border can live around it.
+ * @param highContrastStyle - Style for high contrast mode.
  * @returns The style object.
  */
 export function getFocusStyle(
   theme: ITheme,
   inset: number = 0,
-  position: 'relative' | 'absolute' = 'relative'
+  position: 'relative' | 'absolute' = 'relative',
+  highContrastStyle: IRawStyle | undefined = undefined
```

Comment:
Is there a difference between this and just "IRawStyle?"?

Reply:
Probably not different, but I am getting a tslint error if I don't have the extra bits when I `rush rebuild`:
`missing-optional-annotation  Argument following optional argument missing optional annotation: 
  highContrastStyle: IRawStyle`

Reply:
I can see why it needs to be IRawStyle or undefined, but do you need to set the default to undefined? isn't that what it'll be if it's not set?

Reply:
Alas, that's part of what triggers the error I noted above.